core/src/stages/move_relative.cpp

```diff
@@ -293,35 +293,37 @@ bool MoveRelative::compute(const InterfaceState& state, planning_scene::Planning
 		success = planner_->plan(state.scene(), *link, offset, target_eigen, jmg, timeout, robot_trajectory,
 		                         path_constraints, joint_limits, apply_ruckig_smoothing);
 
-		moveit::core::RobotStatePtr& reached_state = robot_trajectory->getLastWayPointPtr();
-		reached_state->updateLinkTransforms();
-		const Eigen::Isometry3d& reached_pose = reached_state->getGlobalLinkTransform(link) * offset;
-
-		double distance = 0.0;
-		if (use_rotation_distance) {
-			Eigen::AngleAxisd rotation(reached_pose.linear() * ik_pose_world.linear().transpose());
-			distance = rotation.angle();
-		} else
-			distance = (reached_pose.translation() - ik_pose_world.translation()).norm();
-
-		// min_distance reached?
-		if (min_distance > 0.0) {
-			success = distance >= min_distance;
-			if (!success) {
-				char msg[100];
-				snprintf(msg, sizeof(msg), "min_distance not reached (%.3g < %.3g)", distance, min_distance);
-				solution.setComment(msg);
+		if (success) {
```

Comment:
I think it might be better to check `robot_trajectory` since even if the plan is not fully succesfull, it could still be valid depending on what the min_distance is set to
```suggestion
		if (robot_trajectory) {
```

Reply:
Let me test this suggestion to confirm it gives desired behavior